package.json

```
@@ -62,6 +62,21 @@
             "type": "object",
             "title": "Ballerina",
             "properties": {
+                "ballerina.formattingOptions.tabsize": {
+                    "type": "number",
+                    "default": 4,
+                    "description": "Specifies the tab size when formatting ballerina code (Supports for Swan-Lake-Alpha-2 onwards)"
```

Need to remove `-Alpha-2`, Shall we rephrase this as `Supports Swan-Lake onwards`?

---

A suggestion for the wording:
```suggestion
                    "description": "Specifies the tab size when formatting ballerina code (supported on versions following Ballerina Swan Lake)"
```